clouddriver/clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/security/config/AccountsConfiguration.java

```diff
@@ -57,8 +58,8 @@ public static class Account implements AccessControlledAccountDefinition {
     @Include private Boolean enabled;
     @Include private List<CredentialsConfig.Region> regions;
     @Include private List<String> defaultSecurityGroups;
-    private List<String> requiredGroupMembership;
-    @Include private Permissions.Builder permissions;
+    private List<String> requiredGroupMembership = new ArrayList<>();
+    @Include private Permissions.Builder permissions = new Permissions.Builder();
```

Comment:
We don't have a test for this.  The motivator is the `@Nonnull` annotation in [AccessControlledAccountDefinition](https://github.com/spinnaker/spinnaker/blob/9d86a95abc517ea909735f0ee0884bd5f502131d/clouddriver/clouddriver-api/src/main/java/com/netflix/spinnaker/clouddriver/security/AccessControlledAccountDefinition.java#L29).

requiredGroupMembership is the deprecated version of permissions. We didn't see any references to it but decided to give it a default value anyway.

Reply:
Any chance of marking this deprecated too?  I'd love to get rid of that field... :)